core/static/js/opencanada.js

```diff
@@ -1,5 +1,8 @@
 var breakpoint = 1000;
 
+var windowHeight = $(window).height();
```

Comment:
Instead of making these global variables, it probably makes more sense to just access the properties where they are used.